spec/main-spec.js

```diff
@@ -0,0 +1,57 @@
+'use babel'
+
+import Path from 'path'
+import {it, wait} from './helpers'
+
+describe('Main Module', function() {
+  afterEach(function() {
+    waitsForPromise(function() {
+      return atom.packages.uninstallDirectory(Path.join(atom.packages.getPackageDirPaths().pop(), 'auto-semicolon'))
```

Comment:
This line looks rather long, could be split up into multiple. (It's 3 lines in the diff view...)